src/index.ts

```diff
@@ -12,7 +12,7 @@ import * as TS from 'typescript'
 import { loadSync } from 'tsconfig'
 
 const pkg = require('../package.json')
-const shouldDebug = yn(process.env.TS_NODE_DEBUG)
+const shouldDebug = yn(process.env.TS_NODE_DEBUG!)
```

Comment:
Just to get it working for now. Type definitions fro `yn` should be updated instead as environment variables can be missing.